modules/registry-refactoring/src/main/java/org/apache/airavata/registry/core/entities/expcatalog/JobEntity.java

```diff
@@ -21,29 +21,66 @@
 package org.apache.airavata.registry.core.entities.expcatalog;
 
 import javax.persistence.*;
+import java.io.Serializable;
+import java.sql.Timestamp;
 import java.util.List;
 
+/**
+ * The persistent class for the job database table.
+ */
 @Entity
-@Table(name = "EXPCAT_JOB")
-public class JobEntity {
+@Table(name = "JOB")
+public class JobEntity implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @Column(name = "JOB_ID")
     private String jobId;
+
+    @Column(name = "TASK_ID")
```

Comment:
TASK_ID is also part of the primary key.

Reply:
Included @Id and @IdClass